.halcyon/sandbox-extra-apps

```diff
@@ -0,0 +1,3 @@
+alex-3.1.3
+happy-1.19.4
+yesod-bin-1.4.3.1
```

Comment:
It may not be immediately obvious that Halcyon calls itself recursively in order to install extra apps.

For example, when you specify `alex-3.1.3` as a sandbox extra app, somewhere along the process a command similar to `halcyon install alex-3.1.3 --prefix=/app/sandbox` gets executed.